Fix IE11 support (#65)
omgovich/react-colorful: README.md

  </a>
</details>

## Browser support

> It's easier to say that **react-colorful** works everywhere. But if you need more detailed information, we have tested our picker in the browsers that were released in 2014–2015: Chrome 40, Firefox 32, Opera 25, Safari 9 and Edge 12.
> The picker works even in IE11 if an app has an `Object.assign` polyfill installed.
> 

Reviewer: I might reword it like this:

```suggestion
**react-colorful** supports all modern browsers (Chrome, Firefox, Opera, Safari, Edge) and IE11. It will work out of the box with modern browsers and only requires a `Object.assign` polyfill be provided for IE11 support. 
```

I think that saying it "works everywhere" is a bit of a stretch. Unless you're planning on supporting IE6 the scope should probably be limited down to the modern-ish options.

Furthermore I don't think there's a real need to list out what versions are tested against. I think it's incredibly unlikely for a user to be using Firefox or Chrome but using version 20. I think that saying we support the modern browsers is enough detail in that regard.

Author: I want to create a feeling that the picker works not only in modern browsers. That's why I avoided the "modern" word.
I mean when someone opens our README, he will see that our size is less than 2 KB and decided that we support only the latest versions, but that's not true.

Author: I would mention somehow that the picker works even in browsers that were released in 2015. I think that makes developers more confident that they can use our components in their production projects.

Author: Most of the **tiny** frontend projects work only in the modern browsers and don't even support, for example, Edge 15. It would be cool to describe that we support a lot of browsers.

Reviewer: I guess that comes down to reader. I see "modern browser" to mean Chrome, Firefox, etc. regardless of version. Even Chrome 40 is "modern". But I see what you mean, there's no reason to leave that up to interpretation. How's this? 

```suggestion
**react-colorful** aims to support all browsers on the market and we test against legacy versions of many of the popular browsers. **react-colorful** will work out-of-the-box with all modern browsers and only requires that a `Object.assign` polyfill be provided for full IE11 support.
```

Reviewer: > 
> 
> I would mention somehow that the picker works even in browsers that were released in 2015. I think that makes developers more confident that they can use our components in their production projects.

Personally I think listing years support just gets a bit too "down in the details" for what should likely be a high-level overview. Saying exactly what versions are supported, especially with some of these that are on v80+, feels a bit too specific.

Author: Looks good! Thanks ❤️

Reviewer: Or maybe something like this:

```suggestion
It'd be an easier task to list all of the browsers and versions that **react-colorful** does not support! We regularly test against browser versions going all the way back to 2013, and this includes IE11. 

**react-colorful** works out-of-the-box for most browsers, regardless of version, and only requires a `Object.assign` polyfill be provided for full IE11 support. 
```

I think 2013 is a better number due to the IE11 launch date. I doubt anyone's using Firefox or Chrome versions that old.

Reviewer: Bit more of a sales pitch than the previous one

Author: Love the last text a lot!

Author: Exactly what I wanted)
